Assets/Plugins/GameJolt/Scripts/API/Settings.cs

```diff
@@ -20,6 +20,10 @@ public class Settings : ScriptableObject {
 		public bool autoPing = true;
 		[Tooltip("Cache High Score Tables and Trophies information for faster display.")]
 		public bool useCaching = true;
+		[Tooltip("The PlayerPrefs key for the user credentials if the 'Remember Me' option is ticked.")]
+		public string userCredentialsPreferences = "GJ-API-User-Credentials";
```

Comment:
I'm not sure why this would need to be configurable but why not. At least it has a default.

Reply:
sure, it's probably not necessary 😄. It might be better to just use a const string and for the rare case a user wants to change it, he can change the const literal.